apps/masterbots.ai/components/shared/thread-blog-markdown.tsx

```
@@ -0,0 +1,75 @@
+              target="_blank"
+              rel="noopener noreferrer"
+              {...props}
+            >
+              {children}
+            </a>
+          )
+        },
+        code({ node, inline, className, children, ...props }) {
+          //* Handle cursor animation for streaming responses
+          if (children.length) {
+            if (children[0] == '▍')
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)